package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
accidentally included I think?

Reply:
package-lock should be gitignored right?

Reply:
Sorry just to be clear, this package-lock shouldn't be here since it was accidentally created and then included (the real one is in frontend/ folder).

The actual lockfile **needs** to be in git, its the only way to freeze versioning of transitive dependencies (https://github.com/emseers/eelbot-ui/issues/2)